watertap/unit_models/tests/test_gac.py

```diff
@@ -160,34 +131,30 @@ def test_simplified_build(self, gac_frame_simplified):
         port_lst = ["inlet", "outlet", "adsorbed"]
         for port_str in port_lst:
             port = getattr(ms.fs.unit, port_str)
-            assert len(port.vars) == 3  # number of state variables for property package
+            assert len(port.vars) == 3
             assert isinstance(port, Port)
 
         # test statistics
-        assert number_variables(ms) == 100
-        assert number_total_constraints(ms) == 64
-        assert number_unused_variables(ms) == 11  # dens parameters from properties
```

Comment:
It might be worth keeping the comment here (and on line 353) since it may be helpful for a developer that works with this code in the future and is wondering why there are so many unused variables.

Reply:
Issue #950 is concerning diffusivity parameters (and warnings) that are constructed even when unused in this same MCAS property pack. I'll just add a note to this issue to also look at these density parameters to see if they have to be constructed in the first place and re-add the note then if needed.